frontend/src/components/Header/LocForm.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+
+const LocForm = () => {
+  const [loc, setLoc] = useState("");
+  const [city, setCity] = useState("");
+  const [state, setState] = useState("");
+  const [latitude, setLatitude] = useState("");
+  const [longitude, setLongitude] = useState("");
+
+  const handleInputChange = (event) => {
+    setLoc(event.target.value);
+  };
+
+  const handleFormSubmit = async (event) => {
```

Comment:
I suggest also disabling the submit button (even better: displaying a loader) while the request is submitting to avod user spamming the button.
This can be done later in another PR